src/components/AddItems.jsx

```diff
@@ -27,16 +27,19 @@ export function AddItems() {
 			const daysUntilNextPurchase =
 				event.target.elements['purchase-date'].value;
 
-			if (itemName === '') {
-				alert('Please add an item name.');
-				return;
-			}
-			if (!daysUntilNextPurchase) {
-				alert('Please select an option for date');
-				return;
-			}
-
 			try {
+				if (itemName.trim() === '') {
```

Comment:
Was able to confirm that this prevented me from adding a string full of spaces to the list. 🔥